### DISCUSSION TITLE
Trigger Sync on IDLE notifications from host 1

### DISCUSSION THREAD

**Discussion by andyleadbetter** [imapsync/imapsync#427] [Ideas]
I have successfully set this up to sync a google workspace to icloud, and it's working great, nice and robust so far.  One thing I've done is wrap up the call to imapsync with a python script that is logging into the same host1 server, and setting up an IDLE wait in a loop.  When the IDLE completes, I trigger the imapsync and update host2.

It would be great to have this built into imapsync and avoid the extra IMAP connection.

**Comment by gilleslamiral** [comment]
Hi Andy,

I've never played with IDLE.
It looks dedicated to the current selected folder.
Imapsync is multi-folders by default.
You do IDLE in INBOX I guess.
The Imapsync code is not okay to loop, for now, and is relatively far from it, much refactoring is needed.
